tests/csm/rest/test_capacity_quota.py

```diff
@@ -786,20 +790,9 @@ def test_41127(self):
         self.log.info("total objects and size %s and %s ", total_objects, total_size)
         self.log.info("Data size is %s ", data_size)
         self.log.info("Step 4: Perform & Verify GET API to get capacity usage stats")
```

Comment:
Can u correct skip marker for [TEST-40640]

Reply:
done